packages/vscode-common/getExtensionApi.ts

```diff
@@ -1,41 +1,42 @@
 import type {
   CommandServerApi,
+  ExcludableSnapshotField,
+  ExtraContext,
+  ExtraSnapshotField,
+  HatTokenMap,
   IDE,
   NormalizedIDE,
   SerializedMarks,
+  SnippetMap,
   TargetPlainObject,
+  TestCaseSnapshot,
   TextEditor,
 } from "@cursorless/common";
-import type {
-  ExcludableSnapshotField,
-  ExtraContext,
-  ExtraSnapshotField,
-  Graph,
-  TestCaseSnapshot,
-  ThatMark,
-} from "@cursorless/cursorless-engine";
 import * as vscode from "vscode";
 import type { SyntaxNode } from "web-tree-sitter";
-import type { SnippetMap } from "../cursorless-engine/snippets/snippet.types";
-import type { Target } from "../cursorless-engine/typings/target.types";
 
-interface TestHelpers {
-  graph: Graph;
+export interface TestHelpers {
   ide: NormalizedIDE;
   injectIde: (ide: IDE) => void;
 
-  // FIXME: Remove this once we have a better way to get this function
-  // accessible from our tests
-  plainObjectToTarget(
+  hatTokenMap: HatTokenMap;
+
+  commandServerApi: CommandServerApi;
```

Comment:
We now pass along the command server api object, which will be a fake, because we are testing.  Before we got it off the graph